hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeDecommissionManager.java

```diff
@@ -391,17 +391,21 @@ private synchronized boolean checkIfDecommissionPossible(List<DatanodeDetails> d
     int numDecom = dns.size();
     List<DatanodeDetails> validDns = new ArrayList<>(dns);
     int inServiceTotal = nodeManager.getNodeCount(NodeStatus.inServiceHealthy());
+    int unHealthyTotal = nodeManager.getNodeCount(NodeStatus.inServiceStale()) +
+        nodeManager.getNodeCount(NodeStatus.inServiceDead());
     for (DatanodeDetails dn : dns) {
       try {
         NodeStatus nodeStatus = getNodeStatus(dn);
         NodeOperationalState opState = nodeStatus.getOperationalState();
         if (opState != NodeOperationalState.IN_SERVICE) {
           numDecom--;
           validDns.remove(dn);
+          LOG.warn("Cannot decommission " + dn.getHostName() + " because it is not IN-SERVICE");
```

Comment:
IntelliJ's suggestion:
```suggestion
          LOG.warn("Cannot decommission {} because it is not IN-SERVICE", dn.getHostName());
```

Reply:
Done.